ui/app/controllers/jobs/job/index.js

```diff
@@ -31,6 +33,11 @@ export default class IndexController extends Controller.extend(
   sortDescending = false;
   activeTask = null;
 
+  /**
+   * @type {('current'|'historical')}
+   */
+  statusMode = 'current';
```

Comment:
`question`: Is this better off as a boolean?

Reply:
I don't think so; we don't save anything on perf and keeping it a specific string lets us extend this when we want to.

Reply:
Also just from a legibility standpoint, `statusMode=historical` reads cleaner than `statusModeHistorical=true` for an end-user.